lib/aqua/Taskfile.yaml

```diff
@@ -0,0 +1,17 @@
+version: "3"
+
+tasks:
+  install:
+    desc: Installs the Aqua CLI using the universal installer script, sets in PATH, and upgrades.
+    silent: true
+    cmds:
+      - |
+        curl -sSfL -O https://raw.githubusercontent.com/aquaproj/aqua-installer/v2.1.1/aqua-installer
```

Comment:
Should we pass aqua version to the lib as an argument? We can have the latest stable as a default if not set on a higher level.

Reply:
So, yes. BUT the problem is the installer pairs the version with the following line which is the checksum. I don't how to get the install version + the corresponding checksum easily. We could probably do something with this: https://github.com/aquaproj/aqua-installer/blob/main/aqua-checksums.json

But I'm not sure and it just seemed like a lot to try and figure that out for an initial version and since they have the `aqua upgrade` functionality as well.

Reply:
Ah, I see. Mind adding that info to a task summary to keep your findings please (or just a comment would do that)?

Reply:
Will do in a follow up for sure!